packages/@aws-cdk/aws-rds/test/integ.instance.lit.ts

```diff
@@ -74,6 +74,11 @@ class DatabaseInstanceStack extends cdk.Stack {
     // Rotate the master user password every 30 days
     instance.addRotationSingleUser();
 
+    // Rotate additional secret in single-user mode
+    const additionalSecret = new rds.DatabaseSecret(this, 'AdditionalSecret', { username: 'additional-user' });
+    additionalSecret.attach(instance);
+    instance.addRotationSingleUser({ secret: additionalSecret });
```

Comment:
Let's leave the integ test as-is (I think the unit tests are enough here).

Reply:
If you say so 🙂